pkg/reconcilers/base_reconciler.go

```diff
@@ -122,7 +122,7 @@ func (b *BaseReconciler) ReconcileResource(obj, desired common.KubernetesObject,
 }
 
 func (b *BaseReconciler) GetResource(objKey types.NamespacedName, obj common.KubernetesObject) error {
-	b.Logger().Info(fmt.Sprintf("Get object %s", common.ObjectInfo(obj)))
```

Comment:
1) `common.ObjectInfo` returns type/name, not name/namespace. 
2) Indeed, sometimes only the type is empty. I think it only happens with our CRD's. Try to find out why
3) base_reconciler should be consistent. We should log the same for Get, Create, Update, Delete, UpdateStatus

Reply:
When getting the object the user of the method often sends an empty object, where TypeMeta has not been set. I guess it happens because of this

Reply:
See:

```
	existing := batchv1.Job{}
	fmt.Printf("objectinfo: '%s'\n", common.ObjectInfo(&existing))
```

```
objectinfo: '/'
```

Reply:
cc @eguzki

Reply:
I've changed the implementation to get the kind from the object type